modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/PriceTag.java

```diff
@@ -446,24 +431,23 @@ protected void setProductInfo() throws Exception {
 
 	private static final Log _log = LogFactoryUtil.getLog(PriceTag.class);
 
+	private String _additionalDiscountClasses;
+	private String _additionalPriceClasses;
+	private String _additionalPromoPriceClasses;
 	private CommerceContext _commerceContext;
 	private CommerceDiscountValue _commerceDiscountValue;
 	private long _cpDefinitionId;
 	private long _cpInstanceId;
 	private DecimalFormat _decimalFormat;
 	private String _discountLabel;
-	private boolean _displayDiscountLevels;
-	private boolean _displayOneLine;
+	private boolean _displayDiscountLevels = true;
 	private String _formattedFinalPrice;
 	private String _formattedPrice;
 	private String _formattedPromoPrice;
 	private boolean _netPrice = true;
 	private String _promoPriceLabel;
 	private int _quantity;
 	private boolean _showDiscount = true;
-	private boolean _showDiscountAmount;
-	private boolean _showPercentage = true;
-	private boolean _showPriceRange = true;
 	private boolean _showPromo = true;
 	private ThemeDisplay _themeDisplay;
 
```

Comment:
The whole of this Tag should be rolled back to the previous commerce-frontend-taglib version and used that as a starting point to re-accommodate the changes in the JSP templates, with additional integrations if needed.